### PR TITLE
Forms: added validator to addUpload

### PR DESCRIPTION
**Pull request by brabijan**

### CODE REVIEW

**Comment by chemix** [comment]
:+1:

**Comment by Northys** [comment]
👍🏿

File: en/forms.texy [hunk 1]
```diff
@@ -223,6 +223,11 @@ addMultiUpload($name, $label = NULL) .{toc: addMultiUpload()}
 -------------------------------------------------------------
 
 Adds multiple file upload field. This method is available since version 2.2.3. In earlier versions, you can use third optional parameter of `addUpload()` to do the same.
+Validation rules are same as `addUpload()` and adds following:
+
+| `Form::MIN_LENGTH` | minimal uploaded files
```
Comment (dg) [inline en/forms.texy]: Maybe `minimal count` is better

**Comment by dg** [comment]
Thanks

**Pull request merged**